utils/RunONNXModelZoo.py

```diff
@@ -120,7 +120,8 @@ def get_args():
                         metavar='model_name',
                         help="Only process a list of models in the ONNX model zoo."
                         " Passing the name of the models, e.g. 'mnist-8 yolov4'."
-                        " Use -p to know model names.")
+                        " Use -p to know model names. Without -m, the script "
+                        " checks all models in the model zoo.")
```

Comment:
@AlexandreEichenberger adding a comment about using RunONNXModelZoo.py without `-m`.

Reply:
It's very minor, is there a way to only run one specific model? Some models are "XXX" and other are XXX_option"; having the `-m XXX` seems to also trigger the second model. Tx

Reply:
Basically, is there a way to force an exact match.

Reply:
Good catch. It looks like an issue of string matching. Let me fix it with another PR.